src/amphtml-email/20_Samples/Bookmark_Article_Email.html

```diff
@@ -0,0 +1,178 @@
+<!---{
+ "preview": "amp4email",
+ "skipValidation": "true",
+ "draft": "true"
+}--->
+<!--
+## Introduction
+
+This sample showcases how to use AMPHTML email to create an email with a
+personalized digest of articles, giving the user an ability to bookmark some of
+them for later reading.
+-->
+
+<!-- -->
+<!-- Doctype declaration is required. -->
```

Comment:
You don't need to document the boilerplate. Focus on what's specific to this sample.